### DISCUSSION TITLE
[Medium主题] POST_LIST_COVER开关控制封面图片显示

### DISCUSSION THREAD

**Discussion by Final233** [tangly1024/NotionNext#2037] [Ideas]
大佬，能对Medium主题能添加POST_LIST_COVER变量吗
这样就可以实现对Medium主题添加变量控制，让用户使用者可以对文章列表显示图片封面进和开启或者关闭

**Comment by tangly1024** [comment]
已经是支持的，配置位置如下：

https://github.com/tangly1024/NotionNext/blob/bd5ddd30dfdf1ed643c72b029456e257ee68925a/themes/medium/config_medium.js#L6
